src/seth/README.md

```diff
@@ -628,6 +668,12 @@ See also [`seth logs`] which does not decode events.
 
 Reads the current gas price at target chain.
 
+### `seth index`
+
+Show the timestamp of a block (the latest block by default).
+
+    seth age [--block <block>]
```

Comment:
Looks like a lil copy / paste error here...

Reply:
Fixed in e56ceffe